src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -98,10 +98,10 @@ public function load($resource, $type = null)
                 }
             }
 
-            if (1 === substr_count($controller, ':')) {
-                $nonDeprecatedNotation = str_replace(':', '::', $controller);
-                // TODO deprecate this in 5.1
-            }
+            // if (1 === substr_count($controller, ':')) {
+            // $nonDeprecatedNotation = str_replace(':', '::', $controller);
+            // TODO deprecate this in 5.1
+            // }
```

Comment:
???

Reply:
The code and the variable is unused but I commented it to not remove it entirely (to keep the "TODO").

Reply:
what should be deprecated in 5.1? this makes no sense, let's remove from here